tests/test_RFT.cpp

```diff
@@ -4497,6 +4623,204 @@ BOOST_AUTO_TEST_CASE(Segment_Phase_Rates)
+    }
+
+    const auto rft = ::Opm::EclIO::ERft {
+        ::Opm::EclIO::OutputStream::outputFileName(rset, "RFT")
+    };
+
+    const auto xSEG = SegmentResults {
+        rft, "P1", RftDate{ 2000, 1, 2 }
+    };
+
+    BOOST_CHECK_CLOSE(xSEG.ovel( 1), 12.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 2), 11.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 3), 10.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 4),  9.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 5),  8.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 6),  7.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 7),  6.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 8),  5.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel( 9),  4.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel(10),  3.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.ovel(11),  2.0f, 1.0e-5f);
+
+    BOOST_CHECK_CLOSE(xSEG.wvel( 1), 12.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 2), 11.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 3), 10.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 4),  9.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 5),  8.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 6),  7.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 7),  6.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 8),  5.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel( 9),  4.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel(10),  3.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.wvel(11),  2.0f, 1.0e-5f);
+
+    BOOST_CHECK_CLOSE(xSEG.gvel( 1), 12.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 2), 11.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 3), 10.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 4),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 5),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 6),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 7),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 8),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel( 9),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel(10),  0.0f, 1.0e-5f);
+    BOOST_CHECK_CLOSE(xSEG.gvel(11),  0.0f, 1.0e-5f);
+}
+
+BOOST_AUTO_TEST_CASE(Segment_Holdup_Fractions)
+{
+    using RftDate = ::Opm::EclIO::ERft::RftDate;
+
+    const auto rset  = RSet { "TESTSEG" };
+    const auto model = Setup{ segmentDataSet() };
+
+    {
+        auto rftFile = ::Opm::EclIO::OutputStream::RFT {
+            rset, ::Opm::EclIO::OutputStream::Formatted  { false },
+            ::Opm::EclIO::OutputStream::RFT::OpenExisting{ false }
+        };
+
+        const auto  reportStep = 1;
+        const auto  elapsed    = model.sched.seconds(reportStep);
+        const auto& grid       = model.es.getInputGrid();
+
+        ::Opm::RftIO::write(reportStep, elapsed, model.es.getUnits(),
+                            grid, model.sched, wellSol(grid), rftFile);
+    }
+
+    const auto rft = ::Opm::EclIO::ERft {
+        ::Opm::EclIO::OutputStream::outputFileName(rset, "RFT")
+    };
+
+    const auto xSEG = SegmentResults {
+        rft, "P1", RftDate{ 2000, 1, 2 }
+    };
+
```

Comment:
I need some help in figuring out where the difference is in this part to the similar parts in the other tests. Somehow I am missing it.

Reply:
There is no difference in this exact part.  The difference is in the result vectors being tested.

Reply:
Maybe we should make that explicit e.g. by using a common function or [fixture](https://www.boost.org/doc/libs/1_60_0/libs/test/doc/html/boost_test/tests_organization/fixtures/case.html)

Reply:
> Maybe we should make that explicit e.g. by using a common function

I've extracted the setup code to a helper function.  I don't think we need a fixture here since there's no real tear-down code.